common/configloader/configloader.go

```diff
@@ -0,0 +1,173 @@
+package configloader
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+)
+
+var ErrNotSet = errors.New("The value for the given key was not set")
```

Comment:
`const`?

Reply:
`errors.New(...)` isn't evaluated at compile time so it is not `const`.